modules/matterhorn-asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/AssetManagerWithSecurity.java

```diff
@@ -104,9 +159,12 @@ public AssetManagerWithSecurity(
   }
 
   @Override public Opt<Asset> getAsset(Version version, String mpId, String mpeId) {
```

Comment:
What is the difference here between mpId, mpeId mybe Meaningful Variables would help here.